test/extended/images/mysql_replica.go

```diff
@@ -0,0 +1,311 @@
+package images
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/util/wait"
+)
+
+// getPods looks up pods with "oc get" applying a label selector and returns
+// the names of matching pods
+func getPods(oc *exutil.CLI, selector string) ([]string, error) {
```

Comment:
rework this to use `client.PodInterface` instead of exutil.CLI. Then do a client call to get the pods you want.